Assets/01_Scripts/LevelManager.cs

```diff
@@ -4,15 +4,58 @@
 
 public class LevelManager : MonoBehaviour
 {
+    [SerializeField] GameObject GameOverCanvas;
+    [SerializeField] GameObject LevelScoreCanvas;
+
     [SerializeField] private int score = 0;
     [SerializeField] private int lives = 3;
 
+    private float countdownTimeBetweenScreens = 3f;
 
+    private SceneLoader sceneLoader;
```

Comment:
Could you remove this variable? Is only getting initialized on line 33 but never used afterwards